i18n/en.json

```diff
@@ -14,6 +14,12 @@
 	"echo-pref-tooltip-requestssl-request-comment": "Notify me when there is a comment on an SSL certificate request that I have participated on.",
 	"echo-pref-tooltip-requestssl-request-status-update": "Notify me when the status is updated for an SSL certificate request that I have participated on.",
 	"requestssl-button-copy": "(Copy to Clipboard)",
+	"requestssl-cloudflare-comment-active": "This custom domain has been successfully activated and applied to this wiki.",
+	"requestssl-cloudflare-comment-permissions": "An error occurred while processing this custom domain request. The requester is not a bureaucrat so a system administrator will now manually review this request.",
```

Comment:
Suggestion, not requirement:
`An error occurred while processing this custom domain request. Since the requester is not a bureaucrat, a system administrator will manually review it.`